docs/rules/empty-line-between-options.md

```diff
@@ -0,0 +1,121 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/empty-line-between-options
+description: enforce empty lines between top-level options
```

Comment:
For consistency with [vue/padding-line-between-blocks.html](https://eslint.vuejs.org/rules/padding-line-between-blocks.html), [vue/no-restricted-component-options.html](https://eslint.vuejs.org/rules/no-restricted-component-options.html), [vue/no-potential-component-option-typo.html](https://eslint.vuejs.org/rules/no-potential-component-option-typo.html) and [vue/component-options-name-casing.html](https://eslint.vuejs.org/rules/component-options-name-casing.html), I'd suggest:

```suggestion
title: vue/padding-line-between-component-options
description: require or disallow padding lines between top-level component options
```